mantle/platform/platform.go

```diff
@@ -35,7 +35,8 @@ import (
 )
 
 const (
-	sshRetries = 30
+	// Encryption takes a long time--retry more before failing
+	sshRetries = 60
```

Comment:
Wait up to 10min....yuck, but I understand why.

Reply:
I wonder if it wouldn't be better to keep this at the current 5 minute timer and perform the initial SSH connection to the tang server / LUKS machines inside of a Retry that catches the timeout.

Reply:
This was where it was failing for me which is already wrapped in a Retry: https://github.com/coreos/coreos-assembler/blob/42773b9c431921933175a2806158e0b3b7df6545/mantle/platform/journal.go#L119

Reply:
Ah yeah, I thought it was happening on a `c.SSH`/`c.MustSSH` call. Forgot that the `Start` method is chained in the machine creation which would also die out.